--- ExtUtils-Install-2.14/lib/ExtUtils/Install.pm.orig	2017-05-28 12:51:53.000000000 +0200
+++ ExtUtils-Install-2.14/lib/ExtUtils/Install.pm	2017-07-07 21:03:00.526230063 +0200
@@ -825,7 +825,7 @@
                 utime($atime,$mtime + Is_VMS,$targetfile) unless $dry_run>1;
 
 
-                $mode = 0444 | ( $mode & 0111 ? 0111 : 0 );
+                $mode = 0644 | ( $mode & 0111 ? 0111 : 0 );
                 $mode = $mode | 0222
                     if $realtarget ne $targetfile;
                 _chmod( $mode, $targetfile, $verbose );
@@ -1233,7 +1233,7 @@
         }
         my($mode,$atime,$mtime) = (stat $from)[2,8,9];
         utime($atime,$mtime+Is_VMS,$to);
-        _chmod(0444 | ( $mode & 0111 ? 0111 : 0 ),$to);
+        _chmod(0644 | ( $mode & 0111 ? 0111 : 0 ),$to);
         next unless $from =~ /\.pm$/;
         _autosplit($to,$autodir) if defined $autodir;
     }
